src/shared/bus-wait-for-jobs.c

```diff
@@ -200,22 +200,24 @@ static void log_job_error_with_service_result(const char* service, const char *r
 
                 if (i < ELEMENTSOF(explanations)) {
                         log_error("Job for %s failed because %s.\n"
-                                  "See \"%s status %s\" and \"%s -xe\" for details.\n",
+                                  "See \"%s status %s\" and \"%s -xu %s\" for details.\n",
```

Comment:
Some things I was considering;

- would it be useful to add `-n` or `-n NNN`, in case there's many logs?
- while in most cases I encountered (users reporting issues with containerd or dockerd), the `-xe` output is not very useful, perhaps there's situations I did not take into account; if that's the case, perhaps instead of _replacing_ the instructions, print both?

Reply:
adding -u foo seems useful - but why removing -e ? As you mentioned yourself, there might be lots of logs, and printing only the end seems the most useful generic advice

Reply:
Ah, yes, you're right, good suggestion; let me update

Reply:
updated 👍